### PR TITLE
Remove all traces of AB3 in README

### PR DESCRIPTION
**Pull request by voldiemort**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,17 +8,6 @@
...
-    * as a starting point of a course project (as opposed to writing everything from scratch)
-    * as a case study
-  * The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-    * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-    * It comes with a **reasonable level of user and developer documentation**.
-  * It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-  * For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-  * This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing-to-se-edu) for more info.
+* This project, `ContactsForGood (CFG)` is a CLI-based address book targeted towards NGO administrators.
```
Comment (seah-minlong) [inline README.md:11]: Would calling it a CLI-based contact management application designed specifically for NGO administrators be better?

**Review by seah-minlong — APPROVED** [review]
Just a small nit. Is fine if you think its all good.

File: README.md (hunk 1 above)
Comment (voldiemort) [reply README.md:11]: Sounds good, will push the new change soon

**Comment by seah-minlong** [comment]
Closes #69

**Pull request merged**